client/dashboard/utils/site-plans.ts

```diff
@@ -31,19 +31,23 @@ export const isSitePlanBigSkyTrial = ( site: Site ) => {
 };
 
 export const isSitePlanPaid = ( site: Site ) => {
-	if ( ! site.plan ) {
-		return false;
-	}
-
 	return ! [ DotcomPlans.JETPACK_FREE, DotcomPlans.FREE_PLAN ].includes(
-		site.plan.product_slug as DotcomPlans
+		site.plan?.product_slug as DotcomPlans
```

Comment:
This has flipped the logic of `isSitePlanPaid` when `site.plan` is missing. Doesn't it make more sense to return `false` when we don't know what the site's plan is?

Reply:
Yeah you're correct, this change makes the logic less readable. Reverted in 5969f7f.